docs/source/instruction_set_extensions.rst

```diff
@@ -1066,9 +1071,6 @@ performed.
 
 The custom SIMD extensions are only supported if ``PULP_XPULP`` == 1.
 
-**SIMD is not supported by the compiler of the tool chain** as it is not implementing auto-vectorization up to now.
-But those instructions can be used either with builtins or even in assembly.
-
```

Comment:
I think we should keep this comment because it is still true.

Reply:
I will modify it. SIMD will be supported, but we won't have autovectization.